app/classifier/classifier.jsx

```diff
@@ -127,7 +109,7 @@ class Classifier extends React.Component {
   }
 
   getActiveTask(state) {
```

Comment:
not a review comment, just for my understanding, can move this conversation to Slack if prefer, but how is `getActiveTask` used? I don't see it elsewhere in this file, or in the classifier directory.

Reply:
Very good question. I fixed it, but I don't know where it's called either.

Reply:
`getActiveTask` was added in #4191 but isn't actually used anywhere. I've removed it.